lib/msal-browser/src/network/XhrClient.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { INetworkModule } from "msal-common";
+
+export class XhrClient implements INetworkModule {
+
+    async sendGetRequestAsync(url: string, headers?: Map<string, string>, body?: string): Promise<any> {
+        return this.sendRequestAsync(url, "GET", headers, body);
```

Comment:
Can you create an enum for the request types (GET, POST, etc)

Reply:
+1